web/tests/functional/authentication/test_permission_management.py

```diff
@@ -93,7 +93,8 @@ def test_product_permissions(self):
         product_permissions = self._root_auth_client.getPermissions('PRODUCT')
         expected_permissions = [Permission.PRODUCT_ADMIN,
                                 Permission.PRODUCT_ACCESS,
-                                Permission.PRODUCT_STORE]
+                                Permission.PRODUCT_STORE,
```

Comment:
This change can probably be squashed into the original commit that adds the tests,

Reply:
Hmm.. maybe we can, but don't we keep the implementation and tests into separate commits?